app/multitenant/memcache_client.go

```diff
@@ -209,9 +209,14 @@ func (c *MemcacheClient) FetchReports(keys []string) (map[string]report.Report,
 func (c *MemcacheClient) StoreReport(key string, report *report.Report) (int, error) {
```

Comment:
Is there a reason to keep the StoreReport function? Why not inline everything in StoreReportBytes?